lale/operators.py

```diff
@@ -701,6 +793,26 @@ def __getitem__(self, key: str) -> Any:  # enumeration.Enum:
             raise KeyError("No enumeration found for hyper-parameter: " + key)
 
 
+class WGP(object):
```

Comment:
Can you give this class a more descriptive name please? Also, should the class name start with an `_` to indicate it is not part of the public API?